src/assets/assetsUtil.ts

```diff
@@ -95,9 +94,6 @@ export const formatIconUrlWithProxy = ({
   chainId: string;
   tokenAddress: string;
 }) => {
-  let chainIdDec = chainId;
-  if (isHexString(chainId)) {
-    chainIdDec = convertHexToDecimal(chainId).toString();
-  }
-  return `https://static.metaswap.codefi.network/api/v1/tokenIcons/${chainIdDec}/${tokenAddress.toLowerCase()}.png`;
+  const chainIdDecimal = convertHexToDecimal(chainId).toString();
```

Comment:
This is just another instance of `convertHexToDecimal` that I introduced in my last PR for v30.0.1, where we can pull the `isHexString` check into the method itself.